src/APIFunctions/User.js

```diff
@@ -34,6 +34,19 @@ export async function updateLastLoginDate(email, token) {
 }
 
 
+export async function connectToDiscord(email, token) {
+  let status = new UserApiResponse();
+  // eslint-disable-next-line
```

Comment:
why is this here?

Reply:
was because of change previously mentioned